examples/tags/variables.tf

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2023 q.beyond AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "organization_id" {
```

Comment:
Maybe an idea, but using organization domain name instead of id to simplify user input and use a data source in code. Opinion?

Reply:
Mmh i thought about that, but i wanted to have the examples as minimal as possible, therefore i decided against an data source.